components/JournalEntryList/index.js

```diff
@@ -4,44 +4,41 @@ import styled from "styled-components";
 export default function JournalEntryList() {
   return (
     <>
-      <header>
-        <Heading>Your Entries</Heading>
-      </header>
-      <main>
-        <EntryList>
-          {titles.map((title) => (
-            <ListItem key={title.id}>
-              Date: {title.date}
-              <br />
-              Entrytitle: {title.entryTitle}
-              <br />
-              Journalentry: {title.name}
-            </ListItem>
-          ))}
-        </EntryList>
-      </main>
+      <Heading>Your Entries</Heading>
+      <EntryList>
+        {titles.map((title) => (
+          <ListItem key={title.id}>
+            Date: {title.date}
+            <br />
+            Entrytitle: {title.entryTitle}
+            <br />
+            Journalentry: {title.name}
```

Comment:
Using the br tag is not the most charming way. If you want to create a line break, render multiple p tags or choose other elements with semantic value and which are displayed as a block.